hazelcast-client/src/main/java/com/hazelcast/client/spi/impl/CallIdSequenceWithBackoff.java

```diff
@@ -0,0 +1,96 @@
+/*
```

Comment:
Why don't we use the serverside logic for this? I would rather fix this problem once. 

Also we need to pull out the policy from the CallIdGenerator. This way we can offer the old behavior (fail immediately), but also provide an out of the box implementation.